vendor/elv/elv-lw-swd.yaml

```diff
@@ -0,0 +1,120 @@
+name: ELV LoRaWAN® water sensor
```

Comment:
Could you please remove the company name from the name? For the in1 too as well.